### PR TITLE
Add yaml versions of beacon test vectors

### PR DESCRIPTION
**Pull request by acolytec3**
Adds a `yaml` version of the 4 beacon network test vectors

### CODE REVIEW

**Comment by KolbyML** [comment]
Could you remove the json files these yaml files are replacing in this PR as well

File: tests/mainnet/beacon_chain/light_client/optimistic_update.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Optimistic update for slot 6718464
```
Comment (kdeme) [inline tests/mainnet/beacon_chain/light_client/optimistic_update.yaml:1]: Might want to clarify that this is the `signature_slot` as you did in the updates.yaml

File: tests/mainnet/beacon_chain/light_client/finality_update.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Finality update for slot 6718463
```
Comment (kdeme) [inline tests/mainnet/beacon_chain/light_client/finality_update.yaml:1]: Might want to clarify that this is the `attested_header.beacon.slot`

**Review by kdeme — APPROVED** [review]

**Comment by kdeme** [comment]
> Could you remove the json files these yaml files are replacing in this PR as well

I'd argue that is typically better to keep the adding of yaml files and the removal of the json files as separate PRs.
That way, each team can approve the removal PR at their own pace. This would avoid a team needing to urgently fix some yaml/json stuff when other test vectors get added which they want to use.

Anyhow, fluffy is ready to move these to yaml so I don't mind in this specific case to remove them in this PR.

File: tests/mainnet/beacon_chain/light_client/optimistic_update.yaml (hunk 1 above)
Comment (acolytec3) [reply tests/mainnet/beacon_chain/light_client/optimistic_update.yaml]: Done

**Comment by acolytec3** [comment]
> > Could you remove the json files these yaml files are replacing in this PR as well
> 
> I'd argue that it is typically better to keep the adding of yaml files and the removal of the json files as separate PRs. That way, each team can approve the removal PR at their own pace. This would avoid a team needing to urgently fix some yaml/json stuff when other test vectors get added which they want to use.
> 
> Anyhow, fluffy is ready to move these to yaml so I don't mind in this specific case to remove them in this PR.

Okay, will do in a next commit.  Ultralight is likewise ready to move.

**Review by kdeme — APPROVED** [review]

**Review by KolbyML — APPROVED** [review]
:shipit:

**Pull request merged**